packages/core/src/pattern.mjs

```diff
@@ -172,6 +172,12 @@ Pattern.prototype.draftPartForSet = function (partName, set) {
       }
       return result
     } catch (err) {
+      // If the browser's JavaScript engine doesn't include the error's
+      // "name: message" string at the top of the stack, manually push it
+      // onto the stack so the information is shown in the error log.
+      if (!err.stack.startsWith(err.toString())) {
+        err.stack = err.toString() + '\n' + err.stack
+      }
```

Comment:
I'd rather have this happen inside of `log.error()` so that this check is performed in all circumstances rather than just this one case

Reply:
I removed the code from core `pattern.mjs` and moved it into the workbench `logs.js` so all errors are handled.